src/components/ProfilePage/ProfilePage.js

```diff
@@ -0,0 +1,84 @@
+import './profilePage.scss';
+import default_user from '../../assets/images/default-user.jpg';
+import Avatar from './Avatar/Avatar';
+import {useEffect, useState, useRef} from 'react';
+import $ from 'jquery';
+import React from 'react';
+const ProfilePage = (props) =>{
+    const ref_image = React.createRef();
+    const ref_email = useRef(null);
+    const ref_firstName = useRef(null);
+    const ref_lastName = useRef(null);
+    const [user, setUser] = useState([])
+    useEffect(() => {
```

Comment:
Can you please explain the reason for using `useEffect` hook in this case?

Reply:
To retrieve data from the back-end